.github/workflows/ci.yml

```diff
@@ -274,14 +274,6 @@ jobs:
             exit 1
           }
 
-      - name: Install new clang
```

Comment:
should update the matrix labels as well 📅 

_(since clang-14 is already available)_

https://github.com/neovim/neovim/blob/887e935c5e344511d1aec85f116bed558654a219/.github/workflows/ci.yml#L215-L220

Reply:
Hmm, sorry, I didn't follow. Can you elaborate? Do you mean we should bump `clang13` to `clang14`?

Reply:
Or that I should rename `clang` to `clang14` to be more explicit and leave `clang13` label alone?

Reply:
Just write `name: Install clang 13`, I guess, since clang-13 is no longer "new". 

(Pointless bikeshedding, if you ask me.)

Reply:
Oh, that. Yeah sure.

Reply:
No wait what? I've removed that step though, what is there to write?

Reply:
I meant it should just be called `clang`, don't mention the version since it's likely to be 15 soon.

Reply:
But we explicitly set it to clang version 13? Could you use the suggestion feature to show me how you'd like it to look?

Reply:
> But we explicitly set it to clang version 13?

currently, yes that's the case: https://github.com/neovim/neovim/blob/d9dce2d955b0359d1555bfa86aa641f26ad6e8bc/.github/workflows/ci.yml#L245

so I'm suggesting skipping this (arbitrary) restriction, which I assume only existed since `13` was the newest version that the installer supported, see https://github.com/neovim/neovim/commit/6a932f68e6853e4fdcb49621fb1d89829a23238b

Reply:
...that was a roundabout way of explaining that. OK, I understand. I can check if we still need to use clang-13.

Reply:
@zeertzjq do we still need clang 13?